jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -19,7 +19,7 @@ namespace jlm::llvm::aa
 bool
 IsOrContainsPointerType(const rvsdg::Type & type)
 {
-  return IsOrContains<PointerType>(type);
+  return IsOrContains<PointerType>(type) || dynamic_cast<const llvm::FunctionType *>(&type);
```

Comment:
`is<FunctionType>()`

Also the name of the function fits not what it does any longer.

Reply:
I am not sure what to do here. Pointer analysis wants to track this as a pointer since it considers functions and pointers to be the same thing (instead of functions being only pointees). So from its point of view we have to "pretend" that it is a pointer indeed and there is justification for keeping the name.

OTOH if consequently go and rename every place that now may refer to either "a pointer or a function" appropriately, this causes to rename pretty much everything in pointer analysis.

Reply:
Fair point.